ddcz/forms/authentication.py

```diff
@@ -1,9 +1,11 @@
+from datetime import datetime
```

Comment:
Nitpick, but thinking that signup is technically not authentication...so maybe either moving to `signup` or just renaming file to `auth` to align with django app name.